### DISCUSSION TITLE
Links in copyright notice in footer

### DISCUSSION THREAD

**Discussion by pideu-mh** [squidfunk/mkdocs-material#5362] [Q&A] [answered]
It is easily possible to put absolute links in the copyright notice in the footer in `mkdocs.yml`. 

My mkdocs-material-based documentation is a sub-section of some other website, so I used this method to add links for contacts, imprint and privacy which link to the dedicated pages of the "mother homepage". This worked fine.

Now I wanted to have a separate, dedicated `contact.md` in my mkdocs-material documentation, but unfortunately this does not work.

I tried to add a relative link like this:
```
copyright: <a href="contact">&#10140; Contact</a>
```
Initially this seemed to work: clicking on the link takes me to `contact.md` correctly.

However, this only works if the page where the footer is displayed is on "level 1" of the documentation. I have created a minimal working example that uses the following page layout:

```
nav:
  - index.md
  - level2/index.md
  - contact.md
```

If I click on the contact link on the page on "level 2", then I get a 404 error.

I have tried to use `<a href="/contact">` (note the slash) instead, however this does not work with my webserver configuration, because the mkdocs-material pages must be placed in subdirectory and I cannot change the configuration freely.

Is there any way to make this work what I am trying to do?

I have attached a minimal working example that shows the problem I am facing.
[mwe.zip](https://github.com/squidfunk/mkdocs-material/files/11208499/mwe.zip)

**Comment by squidfunk** [comment, marked as answer]
As you noticed, it't not possible to use relative links at this point. What you can do is use [theme extension](https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme), override the [`copyright`](https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/copyright.html) partial and use the `url` filter to make the link relative. Something like:

``` jinja2
<!-- Copyright and theme information -->
<div class="md-copyright">
  {% if config.copyright %}
    <div class="md-copyright__highlight">
      <a href="{{ 'contact' | url }}">&#10140; Contact</a>
    </div>
  {% endif %}
  {% if not config.extra.generator == false %}
    Made with
    <a
      href="https://squidfunk.github.io/mkdocs-material/"
      target="_blank" rel="noopener"
    >
      Material for MkDocs
    </a>
  {% endif %}
</div>
```

**Reply by pideu-mh** [reply]
Thank you very much for your suggestion. I did not know about the `url` filter and actually found little information how it works and what it can be used for.

But your suggestion fixes the problem I was having - thanks again!
